settings.lua

```diff
@@ -2041,7 +2041,10 @@ end
 function SettingList:loadFromXml(xml, parentKey)
 	local value = getXMLInt(xml, self:getKey(parentKey))
 	if value then
+		local tmp = self.syncValue
+		self.syncValue = false -- Make sure there are not network calls during settings loading. Debug channels have not been initialized at that point but the network code contains debug calls
```

Comment:
Any chance we can use the 'noEventSend' pattern Giants is using all over the place? See for example https://gdn.giants-software.com/documentation_scripting_fs19.php?version=script&category=70&class=10571#setPipeState166989